.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+**PULL REQUEST CHECKLIST**
+Please complete the following checklist. Enter y for yes, i for inapplicable before the PR is submitted for review 
+
+- [ ] My pull request has a descriptive title (not a vague title like `Update xyz`)
+* [ ] I have performed a self-review of my own code
+* [ ] I have commented my code, particularly in hard-to-understand areas.
+* [ ] I have documented all fields, methods and types that I added or modified, following [Google style](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html).
+
+If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.
+
+Closes #XXXXX
```

Comment:
Awesome! This is a great idea @dzlab  to have such a checklist.

Since you have done this, I think it is a good occasion that you create a CONTRIBUTING.md file as done in PySyft for instance

https://github.com/OpenMined/PySyft/blob/master/CONTRIBUTING.md

and that this checklist be part of the file.

Reply:
I think for the moment we can just point to this common contributing guidelines. If there are specific guidelines for SyferText then we can add a new CONTRIBUTING.md file.